routes/api.php

```diff
@@ -15,10 +15,26 @@
 |
 */
 
-Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
-    return $request->user();
-});
+// Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
+//     return $request->user();
+// });
 
+// On-boarding Routes
 Route::controller(AuthController::class)->prefix('auth')->group(function () {
-    Route::post('signup', 'signUp')->name('user-signup');
+    Route::post('/register', 'register')->name('user-register');
+    Route::post('/login', 'login')->name('user-login');
+    Route::post('/forgot-pass', 'forgotPassword')->name('user-forgot-password');
+    Route::post('/verify-code', 'verifyCode')->name('user-verify-account');
+});
+
+Route::prefix('user')->middleware("auth:api")->group(function () {
+    Route::controller(AuthController::class)->group(function () {
+        Route::post('/logout', 'logout');
+        Route::post('/reset-password', 'resetPassword');
+    });
 });
+
+
+
+Route::post('/logout',[AuthController::class,'logout'])
```

Comment:
Since we're not using Laravel Sanctum as Authenticating service consider removing this route.